FJ005_geo_mode:
    x_min: -2
    y_min: -2.27
    x_max: 2.6
    y_max: 3.7

FJ005_geo_crit:
    x_min: -1.28
    y_min: -2.00
    x_max: 2.3
    y_max: 3.3

original_x: -1
original_y: -1
original_z: 0.0

take_off1_x: -1
take_off1_y: -1
take_off1_z: 0.5

take_off2_x: -1
take_off2_y: -1
take_off2_z: 1.0

z1_x: 1.5
z1_y: 0.0
z1_z: 1.0

z2_x: 3.0
z2_y: 0.0
z2_z: 1.0


